easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncurses'
+version = '6.0'
+
+homepage = 'http://www.gnu.org/software/ncurses/'
+description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
+ and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
+ function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
```

Comment:
You mean the other way round: I'm gonna drop this one. libreadline and ncurses should be `GCCcore`